pkg/beacon/dkg/dkg.go

```diff
@@ -99,11 +105,17 @@ func ExecuteDKG(
 		}
 	}
 
+	groupOperators := resolveGroupOperators(
+		selectedOperators,
+		gjkrResult.Group.OperatingMemberIDs(),
```

Comment:
Fortunately, this should be easy to fix because v2 smart contract expects the list of misbehaving members [as `uint8[]`](https://github.com/keep-network/keep-core/blob/dc6d086a3a494e2aa3cd56e02d8f2bd3ae69f219/solidity/random-beacon/contracts/libraries/BeaconDkg.sol#L69). I think the way to go is to change `DKGResultSubmission` event to have `Misbehaved     []uint8` instead of `Misbehaved     []byte`, then update `decideMemberFate` to return operating member IDs. We should then either use the original `gjkrResult.Group.OperatingMemberIDs()` or have it overwritten when the fate is decided.

We should also take a look at this loop in `decideFate` and confirm if it does not need to be updated:
https://github.com/keep-network/keep-core/blob/b7d06d305be436fba304f3aac4e432510c7602ee/pkg/beacon/dkg/dkg.go#L159-L168

Reply:
Good catch! See 98f6de88